docs/components/sidecar.md

```diff
@@ -70,7 +71,7 @@ config:
 
 ## Upload compacted blocks (EXPERIMENTAL)
 
-If you want to migrate from a pure Prometheus setup to Thanos and have to keep the historical data, you can use the flag `--shipper.upload-compacted`. This will also upload blocks that were compacted by Prometheus.
+If you want to migrate from a pure Prometheus setup to Thanos and have to keep the historical data, you can use the flag `--shipper.upload-compacted`. This will also upload blocks that were compacted by Prometheus. The `compaction.level` in prometheus blocks marks whether compacted or not, `1` means not compacted.
```

Comment:
Nit: capitalize Prometheus
Also consider:

Values greater than `1` in the `compaction.level` field of a Prometheus block’s meta.json file indicate compaction.